src/Tabs/Tab.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useContext, HTMLAttributes, useCallback } from 'react';
+import styled from 'styled-components';
+import { TabContext, ContextProps } from './Tabs';
+
+const TabComponent = styled.div`
+  margin-right: 39px;
+  display: flex; 
+  align-items: center;  
+`;
+
+const TabLabel = styled.label<{ active: boolean }>`
+  height: 40px;
+  font-family: Monorale;
```

Comment:
Moving forward we are trying to use the theme to set fonts, so new work ideally calls fonts from there (even though there are instances that still need updating to this). 

For setting a font try using:
```
${({theme}) => css`
    font-family: ${theme.fontFamily.ui};
`}
```
There are different ways to write this but this will give you the idea. Using this pattern means you can add in other things too. Theme will be available by default in this context so as long as this is in the Styled Component, it should be good to use!

For reference:
`theme.fontFamily.ui` will be Monorale.
`theme.fontFamily.data` will be Lato.

This means in future projects where fonts may change, we can handle this really easily. It's not enough to block a merge but definitely a big nice to have.

Reply:
Done